app/containers/AppContainer.js

```diff
@@ -19,17 +15,15 @@ import {
   Button
 } from 'react-native';
 
+const path = 'https://movied.herokuapp.com/popular';
```

Comment:
Unused constant? This shouldn't be here. Try to put this on the actions or the middleware.